app/ai/voice/agents/breeze_buddy/stt/__init__.py

```
@@ -60,62 +178,138 @@ async def get_stt_service(
+                        if language_hints is not None
+                        else BREEZE_BUDDY_SONIOX_LANGUAGE_HINTS
+                    ),
+                    context_json=effective_context,
+                    enable_non_final_tokens=BREEZE_BUDDY_SONIOX_ENABLE_NON_FINAL_TOKENS,
+                    max_non_final_tokens_duration_ms=BREEZE_BUDDY_SONIOX_MAX_NON_FINAL_TOKENS_DURATION_MS,
+                    max_endpoint_delay_ms=BREEZE_BUDDY_SONIOX_MAX_ENDPOINT_DELAY_MS,
+                    log_context="Breeze Buddy",
+                    language_hints_strict=True if language_hints else False,
+                    # Disable auto-reconnect when fallback is enabled so a single
+                    # error triggers an immediate swap to Deepgram instead of
+                    # retrying 3× (12 s dead air) and then emitting a fatal
```

Ruff (0.15.5)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)